Library/Homebrew/tap.rb

```diff
@@ -801,7 +802,9 @@ def formula_renames
     end
   end
 
-  # Hash with tap formula old names. Reverse of {#formula_renames}.
+  # Mapping from new to old formula names. Reverse of {#formula_renames}.
+  #
+  # @private
```

Comment:
I don't know if YARD supports this but: I'd rather we flipped this at some point so we assume literally everything that's not declared to be a public API is a private one.